integration-tests/package.json

```diff
@@ -15,6 +15,7 @@
     "test:integration": "hardhat --network optimism test",
     "test:integration:live": "IS_LIVE_NETWORK=true hardhat --network optimism-live test",
     "test:sync": "hardhat --network optimism test sync-tests/*.spec.ts --no-compile",
+    "test:load": "hardhat --network optimism test load-tests/*.spec.ts --no-compile",
```

Comment:
Maybe `test:concurrency`? We should be careful to not overload the term "load"

Reply:
Instead just made this file part of the standard integration tests.